AdminUi/apps/admin_ui/lib/pages/login_page.dart

```diff
@@ -34,56 +38,128 @@ class _LoginScreenState extends State<LoginScreen> {
 
   @override
   Widget build(BuildContext context) {
+    final screenSize = MediaQuery.of(context).size;
```

Comment:
never (ever!!!!!) use `MediaQuery.of(context)` - the way we do this is `MediaQuery.sizeOf(context)`